### ISSUE TITLE
Question about types to use

### ISSUE THREAD

**Issue by Ofer-Gal** [aralroca/fragmented-store#1]
Can this be used with complex objects? not just strings and numbers? 
Arrays of objects?

**Comment by aralroca** [comment]
Yes sure, you can try it here https://codesandbox.io/s/fragmented-store-example-4p5dv?file=/src/App.js 

However,  at the moment, fragmentation is only done at the first level.
